webside/views.py

```diff
@@ -27,63 +27,46 @@ def home():
 
-# tym sposobem udalo sie zachowac funkcjonalnosc checkboxsow zamiast dodawac przycisk do kazdej notatki
-@views.route('/send-or-download-note', methods=['POST'])
-def send_or_download_note():
-    action = request.form.get('action')
-    selected_notes = request.form.getlist('selected_notes[]')
+    return redirect(url_for('views.home'))
 
-    if action == 'send':
-        return redirect(url_for('send_notes', selected_notes=','.join(selected_notes)))
-    elif action == 'download':
-        return redirect(url_for('download_pdf', selected_notes=','.join(selected_notes)))
-    else:
-        return jsonify({'error': 'Invalid action'})
 
-@views.route('/send-note/', methods=['GET', 'POST'])
+@views.route('/send-note/', methods=['POST'])
 def send_notes():
-    if request.method == 'POST':
-        selected_notes = request.args.get('selected_notes').split(',')
-        user_name = current_user.first_name
-        subject = f'Here are your notes, {user_name}'
-        user_mail = current_user.email
+    selected_notes = request.args.get('selected_notes[]').split(',')
+    user_name = current_user.first_name
+    subject = f'Here are your notes, {user_name}'
+    user_mail = current_user.email
 
-        body = join_notes(selected_notes, joiner="\n\n")
+    body = join_notes(selected_notes, joiner="\n\n")
 
-        # tu jest cos zjebane z konfiguracja serwera poczty, ale chyba przez to ze nie chcialem im zaplacic
-        with smtplib.SMTP('live.smtp.mailtrap.io', 587) as smtp:
-            smtp.login('api', 'e89b47ed2b2954dfb93f46e0444dd8ce')
-            mail_contents = f'Subject: {subject}\n\n{body}'
+    # tu jest cos zjebane z konfiguracja serwera poczty, ale chyba przez to ze nie chcialem im zaplacic
+    with smtplib.SMTP('live.smtp.mailtrap.io', 587) as smtp:
+        smtp.login('api', 'e89b47ed2b2954dfb93f46e0444dd8ce')
+        mail_contents = f'Subject: {subject}\n\n{body}'
 
-            smtp.sendmail('mailtrap@demomailtrap.com', user_mail, mail_contents)
-
-        flash('Notes sent!', category='success')  # to dziala dopiero po odswiezeniu i nie wiem czemu
+        smtp.sendmail('mailtrap@demomailtrap.com', user_mail, mail_contents)
 
+    flash('Notes sent!', category='success')  # to dziala dopiero po odswiezeniu i nie wiem czemu
     return jsonify({})
 
-@views.route('/download-note/', methods=['GET', 'POST'])
+
+@views.route('/download-note/', methods=['POST'])
 def download_pdf():
-    global pdf_path
-    if request.method == 'POST':
-        selected_notes = request.args.get('selected_notes').split(',')
-        body = join_notes(selected_notes, joiner="<br/><br/>")
+    selected_notes = request.form.getlist('selected_notes[]')
+    body = join_notes(selected_notes, joiner="<br/><br/>")
 
-        # notatki do pliku html
-        file_html = open("notatki1.html", "w")
+    html_file_name = 'notatki1.html'
+    # notatki do pliku html
+    with open(html_file_name, 'w') as file_html:
```

Comment:
na to zwróć uwagę, pliki powinno się otwierać w 99% procentach za pomocą `with open('filename') as file`